### PR TITLE
FIX: Correctly handle Philips DICOMs w/ derived volume

### LINKED ISSUE #693: Exclude trace volume from Philips DICOM?
Originally brought up in nipy/heudiconv#275 by @bmelo

Version: `nibabel-2.3.1`

Loading up a Philips diffusion DICOM (with trace volume) results in an incorrect image shape with an extra? dimension (96, 96, 60, 2, 33).

```python
In [1]: from nibabel.nicom.dicomwrappers import wrapper_from_file

In [2]: mw = wrapper_from_file('philips-dwi.dcm', force=True)

In [3]: mw.image_shape
---------------------------------------------------------------------------
WrapperError                              Traceback (most recent call last)
<ipython-input-3-4963e90423f3> in <module>()
----> 1 mw.image_shape

/home/mathias/miniconda2/envs/dev3/lib/python3.6/site-packages/nibabel/onetime.py in __get__(self, obj, type)
    139         # Errors in the following line are errors in setting a
    140         # OneTimeProperty
--> 141         val = self.getter(obj)
    142 
    143         setattr(obj, self.name, val)

/home/mathias/miniconda2/envs/dev3/lib/python3.6/site-packages/nibabel/nicom/dicomwrappers.py in image_shape(self)
    535         n_vols = np.prod(shape[3:])
    536         if n_frames != n_vols * shape[2]:
--> 537             raise WrapperError("Calculated shape does not match number of "
    538                                "frames.")
    539         return tuple(shape)

WrapperError: Calculated shape does not match number of frames.
```

### PR DESCRIPTION
**Pull request by effigies**
Rewrite of #727 using a data submodule. Paths currently still assume the file is directly in nibabel.

cc @mgxd

Fixes #693.

### CODE REVIEW

File: nibabel/nicom/tests/test_dicomwrappers.py [hunk 1]
```diff
@@ -36,6 +36,7 @@
 DATA_FILE_DEC_RSCL = pjoin(IO_DATA_PATH, 'decimal_rescale.dcm')
 DATA_FILE_4D = pjoin(IO_DATA_PATH, '4d_multiframe_test.dcm')
 DATA_FILE_EMPTY_ST = pjoin(IO_DATA_PATH, 'slicethickness_empty_string.dcm')
+DATA_FILE_4D_DERIVED = pjoin(IO_DATA_PATH, '4d_multiframe_with_derived.dcm')
```
Comment (effigies) [inline nibabel/nicom/tests/test_dicomwrappers.py]: Need to update this to find data in the submodule.

File: nibabel/nicom/tests/test_dicomwrappers.py [hunk 2]
```diff
@@ -622,6 +623,13 @@ def test_slicethickness_fallback(self):
         dw = didw.wrapper_from_file(DATA_FILE_EMPTY_ST)
         assert_equal(dw.voxel_sizes[2], 1.0)
 
+    @dicom_test
```
Comment (effigies) [inline nibabel/nicom/tests/test_dicomwrappers.py]: Need to add check for submodule.

**Comment by mgxd** [comment]
ah - I was working on this but looks like you beat me to the punch

**Comment by effigies** [comment]
Ah, sorry. I also just updated the test locations...

**Comment by effigies** [comment]
@mgxd Care to re-review and merge if tests pass?

**Pull request merged**